src/components/Tile.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Tile({children}) {
+    return (
+        <section>
+            {children}
```

Comment:
Als enkel de property {children} word gebruikt is er wellicht geen meerwaarde om hier een component van te maken omdat de children toch al uitgeschreven worden. Al was je hier wellicht nog mee bezig :)